pkg/analysis/analysis.go

```diff
@@ -157,13 +207,26 @@ func (a *Analysis) RunAnalysis() {
 		AIClient:      a.AIClient,
 		OpenapiSchema: openapiSchema,
 	}
+	analyzerConfig.Resources = make(map[string][]string)
+	for k, v := range resourceMap {
+		analyzerConfig.Resources[k] = v
+	}
+	if len(analyzerConfig.Resources) > 0 && len(analyzerConfig.Namespace) == 0 {
```

Comment:
Why not always set it to "default" on empty? Also comment is probably not needed here.

Reply:
nvm, I am wrong here.